docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx

```diff
@@ -428,7 +428,16 @@ After the above conditions are satisfied, you can start to configure a multi-clu
 #### Set up Multi-Cluster Replication prior to v1.14
 
 You can set this up with [`clusterMetadata` configuration](/references/configuration#clustermetadata); however, this is meant to be only a conceptual guide rather than a detailed tutorial.
-Please reach out to us if you need to set this up.
+
+:::tip
+
+If you need help when setting up, please reach out to our [community Slack](https://temporalio.slack.com).
+Good places to start include searching through previous conversations and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
```

Comment:
How about suggesting a channel? Reaching out to the community Slack might be a bit vague. Maybe the #support-community would be a good suggestion.